### ISSUE TITLE
how to get loss, val loss, and accuracys from tensorflow with hyperdash

### ISSUE THREAD

**Issue by copypasteearth** [hyperdashio/hyperdash-sdk-py#146]
hi im trying to figure out how to get the losses and accuracies using tensorflow into hyperdash. I can see the console logs fine but theres no metrics
